src/parser/Tokenizer.h

```diff
@@ -907,11 +780,10 @@ class Tokenizer {
   void skipWhitespace() {
     auto v = view();
     auto pos = v.find_first_not_of("\x20\x09\x0D\x0A");
-    if (pos != string::npos) {
-      _data.remove_prefix(pos);
+    if (pos == string::npos) {
+      pos = _data.size();
     }
```

Comment:
I don't think it would have a significant cost (relative to the rest of the parsing) if we increase a counter here every time we encounter a newline. This would be very valuable for the parser error messages.

Reply:
This also is an additional feature, which will be done in connection with better parsing errors.